### PR TITLE
Cmc corr start config

### PR DESCRIPTION
**Pull request by mmphego**
I made all necessary changes, in order to start correlator from test script

### CODE REVIEW

File: mkat_fpga_tests/__init__.py [hunk 1]
```diff
@@ -25,6 +30,18 @@ def __init__(self, test_config_filename=None):
         self._correlator = None
         self._dhost = None
 
+        self.io_manager = ioloop_manager.IOLoopManager()
```
Comment (brickZA) [inline mkat_fpga_tests/__init__.py]: I'll merge this for now, but generally side-effects (such as connecting to network devices) in an __init__ it not a good idea.

We also need to figure out how to do proper cleanup.

File: mkat_fpga_tests/__init__.py [hunk 2]
```diff
@@ -65,84 +82,85 @@ def correlator(self):
             LOGGER.info('Correlator started succesfully')
             return self._correlator
 
-    def start_stop_data(self, start_or_stop, modes):
-        self.modes = modes
-        LOGGER.info('Correlator starting to capture data.')
-        assert start_or_stop in ('start', 'stop')
 
+    def start_stop_data(self, modes):
```
Comment (brickZA) [inline mkat_fpga_tests/__init__.py]: What are 'modes'? The CBF only does instruments and products, not modes.

File: mkat_fpga_tests/__init__.py [hunk 3]
```diff
@@ -65,84 +82,85 @@ def correlator(self):
...
-        self.start_stop_data('stop', 'c856M4k')
+        LOGGER.info ('Stop X data capture')
+        self.katcp_rct.req.capture_stop(self.output_product)
 
     def start_correlator(self, retries=30, loglevel='INFO'):
         success = False
         retries_requested = retries
-        array_no = 0
-        self.dhost
+        self.dhost # starting d-engine
```
Comment (brickZA) [inline mkat_fpga_tests/__init__.py]: You're not doing anything with self.dhost?

Comment (mmphego) [inline mkat_fpga_tests/__init__.py]: If I don't call it, D-Engine does not get initialised.

File: mkat_fpga_tests/__init__.py (hunk 2 above)
Comment (mmphego) [inline mkat_fpga_tests/__init__.py]: Redundant function, Deleted.

**Pull request merged**